diff -urN etherboot-4.2.orig/src-32/Config etherboot-4.2/src-32/Config
--- etherboot-4.2.orig/src-32/Config	Mon Aug  2 21:14:25 1999
+++ etherboot-4.2/src-32/Config	Mon Aug  2 21:21:05 1999
@@ -93,7 +93,8 @@
 
 CC=		gcc
 LD=		ld
-CFLAGS+=	-O2 -fstrength-reduce -fomit-frame-pointer -m386
+CFLAGS+=	$(RPM_OPT_FLAGS)
+CFLAGS+=	-fstrength-reduce -fomit-frame-pointer 
 CFLAGS+=	-Wall -Wno-format -Wno-unused
 ASFLAGS=
 AS86FLAGS=
