diff -urN etherboot-5.0.0.org/src/Config etherboot-5.0.0/src/Config
--- etherboot-5.0.0.org/src/Config	Wed May  2 11:12:05 2001
+++ etherboot-5.0.0/src/Config	Wed May  2 11:13:36 2001
@@ -245,5 +245,5 @@
 CFLAGS32+=	-Wall -W -Wno-format -Wno-unused
 ASFLAGS32=
 LDFLAGS32+=	-N -Ttext $(RELOCADDR) -e _start
-LDBINARY32=	-oformat binary		# flag for headerless binary
+LDBINARY32=	--oformat binary		# flag for headerless binary
 LIBC32=					# not needed
